diff --git i/mkspecs/devices/linux-rpi3-g++/qmake.conf w/mkspecs/devices/linux-rpi3-g++/qmake.conf
index 10862ccd70..edaf48a92b 100644
--- i/mkspecs/devices/linux-rpi3-g++/qmake.conf
+++ w/mkspecs/devices/linux-rpi3-g++/qmake.conf
@@ -8,13 +8,13 @@ include(../common/linux_device_pre.conf)
 # and possibly no pkg-config, have some static values as well:
 
 # I consider it a bug that this is required, but our EGL config.test _requires_ it
-QMAKE_LFLAGS           += -Wl,-rpath-link,$$[QT_SYSROOT]/opt/vc/lib
+QMAKE_LFLAGS           += -Wl,-rpath-link,$$[QT_SYSROOT]/usr/lib/vc
 
-VC_LIBRARY_PATH         = /opt/vc/lib
-VC_INCLUDE_PATH         = =/opt/vc/include
+VC_LIBRARY_PATH         = /usr/lib/vc
+VC_INCLUDE_PATH         = =/usr/include/vc
 
 # terrible, they do not appear to resolve "=" in rpath!
-VC_LINK_LINE            = -L=$${VC_LIBRARY_PATH} -Wl,-rpath-link,$$[QT_SYSROOT]$${VC_LIBRARY_PATH}
+VC_LINK_LINE            = -L$${VC_LIBRARY_PATH} -Wl,-rpath-link,$$[QT_SYSROOT]$${VC_LIBRARY_PATH}
 
 QMAKE_LIBDIR_OPENGL_ES2 = =$${VC_LIBRARY_PATH}
 QMAKE_LIBDIR_EGL        = $$QMAKE_LIBDIR_OPENGL_ES2
